src/app.js

```diff
@@ -322,11 +327,11 @@ class Messenger extends EventEmitter {
   //////////
 
   getCacheKey(senderId/*: number */)/*: string */ {
-    return `${config.get('facebook.appId')}-${senderId}`;
+    return '' + senderId;
```

Comment:
It may be premature optimization, but I think it's best practice to add a layer of abstraction to the key finder, mostly for versions or hash functions.

We could go the other way too and just document that the cache key is the `senderId`.